<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgRatingDlg</class>
 <widget class="QDialog" name="DlgRatingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rbtnScore1">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../res/images/controllers/1stars.png</normaloff>../../res/images/controllers/1stars.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnScore2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../res/images/controllers/2stars.png</normaloff>../../res/images/controllers/2stars.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnScore3">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../res/images/controllers/3stars.png</normaloff>../../res/images/controllers/3stars.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnScore4">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../res/images/controllers/4stars.png</normaloff>../../res/images/controllers/4stars.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnScore5">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../res/images/controllers/5stars.png</normaloff>../../res/images/controllers/5stars.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Please rate for current preset, Thank you ;)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgRatingDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgRatingDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
